### ISSUE TITLE
Sitewide Content/IA Review from 10-10 Health Apps Team [Household section]

### ISSUE THREAD

**Issue by markfallows** [department-of-veterans-affairs/va.gov-team#48245]
## What does your team need support for? Check all that apply.
- [x] Launching one or more new unauthenticated (static) VA.gov pages
- [x] Making revisions to one or more existing unauthenticated (static) VA.gov page
- [ ] Launching a new online form, app, or tool on VA.gov
- [x] Making revisions to an existing online form, app, or tool on VA.gov
- [ ] Something else (please add details in the next question)

## What do you need help with?
  - [x] Content support
  - [x] IA support
  
  
The Household Information section of the 10-10EZ is the area where we have the largest dropoff.  This is most likely due to the information required and the way requests for that information are presented.  To help address this, we're attempting to restructure the section to make it easier to approach.  This restructuring largely involves the reorganization of the information by grouping questions that are similar to one another to help create a more fluid experience; and by ensuring that there's enough information readily available to help Veterans complete the application.

### Content support focus

The household section of the form has already established content, and updated content. The updated content to focus on is described here:
- Headings and body paragraph content on the [first page of the household section](https://www.sketch.com/s/da85cf44-4503-4e98-834e-ff068b242ef6/a/ygvYdEq)
- The sub-labeling, and expandable info on the ["Your annual income"](https://www.sketch.com/s/da85cf44-4503-4e98-834e-ff068b242ef6/a/L0yqdk5) page
- Labeling on the [dependent's additional information page](https://www.sketch.com/s/da85cf44-4503-4e98-834e-ff068b242ef6/a/jgwqgl7)
- Labeling on the ["Previous calendar year's deductible expenses"](https://www.sketch.com/s/da85cf44-4503-4e98-834e-ff068b242ef6/a/8yKwlVO) page

## Will this new product be released incrementally (for instance 25% of users initially)? 
  - [x] Yes
  - [ ] No
 
**Note:** If you checked yes, we'll reach out to discuss details about the content in the react widget.
  
## When do you expect to launch your product to 100% of users?
  
The current estimated release period is early- to mid-January.
  
## Supporting artifacts

- [Link to product outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/application/va-application/Household%20section%20redesign/ReadMe.md)
- [Link to or attach user flows](https://www.sketch.com/s/da85cf44-4503-4e98-834e-ff068b242ef6/a/nR0pl7W)
- [Link to prototype or mockups](https://www.sketch.com/s/da85cf44-4503-4e98-834e-ff068b242ef6/p/208CCE1E-DC03-457F-B2FA-1232663983CF/canvas)

## Will this work be going through the Collaboration Cycle?
- [x] Yes 
- [ ] No

## When does this work need to be done?
- Estimated launch date: 02/06/2023
- Estimated staging review date: 01/09/2022
- Content and IA work needed by: 11/07/2022

## Do you plan to bring this to an upcoming content office hours session?
- [ ] Yes, benefit content office hours (Thursdays, 3:00 p.m. to 3:45 p.m. ET)
- [ ] Yes, health  content office hours (Thursdays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] Yes, unauth office hours (Mondays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] No, but I'd like to schedule time to talk about this request
- [x] No, let's work asynchronously and meet if needed

**Note:** If we think this work would benefit from a collaborative session with you, we may ask you to bring it to office hours or set up a separate time to meet.

## About your team

- Team name: 10-10 Health Apps team
-	OCTO-DE product owner: Patrick Bateman
-	Product manager: Heather Justice, Mark Fallows
-	Designer: Nick Osmanski, Jessica Stump
-	FE engineer: Lisa Zapson, Matt Long
-	Product/team Slack channel: 1010-health-apps

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#sitewide-content-ia](https://dsva.slack.com/channels/sitewide-content-ia) Slack channel and tag Randi Hecht.**

If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=ia&template=redirect-request.md&title=Redirect+Request)

**Comment by heyjoni** [comment]
IA feedback:

Change(s):
- H1 should be "Apply for VA health care" to match the breadcrumb

OK:
- Not having a left nav for the form flow is correct
- Not having breadcrumbs for steps within the form is correct

**Comment by nicko-amida** [comment]
@heyjoni [H1 headings have been update across all pages](https://www.sketch.com/s/da85cf44-4503-4e98-834e-ff068b242ef6/a/ygvYdEq) - I'm checking with @DanielleThierryUSDSVA to see if there's any additional content suggestions still being worked on.

**Comment by DanielleThierryUSDSVA** [comment]
Hi @nicko-amida! 

I added some comments to the sketch file. Question for you: The annual income and dependent information links both gave me a "This artboard doesn't exist" message. I started to just find those screens from the other links, but I'm not sure they're the ones you want me to review because the ticket references "expandable info" on the "Your annual income" page and I don't see any expandable info on that screen. So I wanted to check on this before I comment further. Thoughts?

Also, I know that most of the content in this section is existing content, but I think we should review it all together at some point soon. The 10-10EZ is an older application and I don't think I've ever reviewed this section and I noticed at least a few things we can update. I'm happy to do the first level review now and then review the full section together when we have time. Thanks!

**Comment by nicko-amida** [comment]
@DanielleThierryUSDSVA Hey! I just updated the links so they should work - some got broken as I made updates from other collab cycle tickets. I also removed the "expandable info" components on those pages, so you can ignore that now. Just the label/sub-label content is fine! And sure, if we need to do a whole section review beyond the pages I listed that's cool.

**Comment by nicko-amida** [comment]
@DanielleThierryUSDSVA Hey quick update - I already made changes based on your comments, but I just went through to update my replies in the [Sketch pages](https://www.sketch.com/s/da85cf44-4503-4e98-834e-ff068b242ef6/a/ygvYdEq).

**Comment by DanielleThierryUSDSVA** [comment]
Thanks, @nicko-amida! We can talk more about reviewing this from an overall perspective at our next meeting, which I think won't be until after Thanksgiving. Hope you have a good weekend this week, and a good long holiday weekend next week!

**Comment by jessicastump** [comment]
I just found this ticket and wanted to check and see if it should be closed or not. cc @DanielleThierryUSDSVA, @aliyahblackmore 
This was used during the design intent to midpoint phases when getting a prototype ready for a research study. That study was completed (#192) and the findings are being incorporated into the design and for staging review. We opened [this content review ticket #52920](https://github.com/department-of-veterans-affairs/va.gov-team/issues/52920) for that current work.
